spec/index.html

```diff
@@ -5669,248 +5731,221 @@ <h5>NOT IN</h5>
+                <a data-cite="RDF12-CONCEPTS#dfn-triple-term-equality">equal as triple terms</a>; that
+                is, the
+                <a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a>,
+                <a data-cite="RDF12-CONCEPTS#dfn-predicate">predicate</a>, and
+                <a data-cite="RDF12-CONCEPTS#dfn-object">object</a> components are pair-wise
+                the same term.
+              </li>
+            </ul>
+            <div class="result">
+              <table>
+                <tbody>
+                  <tr>
+                    <td><code>sameTerm(&lt;http://example/&gt;, &lt;http://example/&gt;)</code></td>
+                    <td>true</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm(&lt;http://example/&gt;, &lt;https://example/&gt;)</code></td>
+                    <td>false</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm("abc", "abc")</code></td>
+                    <td>true</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm("abc"@en, "abc")</code></td>
+                    <td>false</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm("abc"@en, "abc"@EN)</code></td>
+                    <td>true</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm("abc"@en--rtl, "abc"@en)</code></td>
+                    <td>false</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm(2, 2.0)</code></td>
+                    <td>false</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm(2, "2"^^xsd:integer)</code></td>
+                    <td>true</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm(2, "02"^^xsd:integer)</code></td>
+                    <td>false</td>
+                  </tr>
+              </table>
+            </div>
+
+          </section>
+          <section id="func-sameValue">
+            <span id="func-RDFterm-equal"><!-- obsolete id --></span>
+            <h5>sameValue</h5>
+            <pre class="prototype nohighlight">
+              <span class="return">xsd:boolean</span> <span class="operator" style="text-transform: none;">sameValue</span> (<span class="type"><span class="type RDFterm">RDF term</span></span> <span class="name">term1</span>, <span class="type"><span class="type RDFterm">RDF term</span></span> <span class="name">term2</span>)
+            </pre>
+
+            <p>This function cannot be used directly in expressions. The purpose
+              of this function is to define the semantics of the "=" operator when applied to
+              two RDF terms that do not fall into any of the other, more concrete cases
+              covered in the operator mapping table in Section
```

Comment:
Formulation is a bit strange here. As it could be interpreted as there being a category of "other" cases within the operator table.
```suggestion
              two RDF terms that do not fall into the concrete cases
              covered in the operator mapping table in Section
```